core/bin/zksync_api/src/fee_ticker/tests.rs

```diff
@@ -677,3 +678,49 @@ async fn test_error_api() {
         .await
         .unwrap();
 }
+
+#[tokio::test]
+#[ignore]
+async fn test_rdoc_price() {
+    const DATABASE_URL: &str = "postgres://postgres@localhost/plasma";
+    const BASE_URL: &str = "http://127.0.0.1:9876";
+    const RDOC_SYMBOL: &str = "RDOC";
+
+    let client = reqwest::ClientBuilder::new()
+        .timeout(CONNECTION_TIMEOUT)
+        .connect_timeout(CONNECTION_TIMEOUT)
+        .build()
+        .expect("Failed to build reqwest::Client");
+
+    env::set_var("DATABASE_URL", DATABASE_URL);
```

Comment:
Im not sure about this, I got the same error but just when tried to run it directly, but by running it with the zk command, it works properly.